# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome14/gtk-perl10/gendefs.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./tools/gendefs.pl~	Tue Mar  9 19:46:14 1999
+++ ./tools/gendefs.pl	Tue Apr 27 16:08:13 1999
@@ -517,6 +517,10 @@
 #undef _
 #endif
 
+#ifdef dirty
+#undef dirty
+#endif
+
 #ifndef Perl${FilePrefix}DeclareFunc
 #include "Perl${FilePrefix}Int.h"
 #endif
